test/migrations.test.js

```diff
@@ -0,0 +1,228 @@
+process.env.NODE_ENV = 'test';
+
+const assert = require('chai').assert;
+const { suite, test } = require('mocha');
+const knex = require('../knex');
+const { addDatabaseHooks } = require('./utils');
+
+suite('migrations', addDatabaseHooks(() => {
+  test('user_map columns', (done) => {
+    knex('user_map').columnInfo()
+      .then((actual) => {
+        const expected = {
+          id: {
+            type: 'integer',
+            maxLength: null,
+            nullable: false,
+            defaultValue: 'nextval(\'user_map_id_seq\'::regclass)',
```

Comment:
I removed `::regclass` and it broke all of my tests so it seems like we do have to use it. I have done some extensive research on the matter and still don't fully understand it but it seems like it has something to do with using the `primary()` attribute in knex.js. I'm happy to continue researching this until I fully understand it or just replace the `::regclass` and move on. I'm attaching a screenshot of an example of a query for 'message' column names and data types. In the 'Modifiers' column for the first row, 'id', you can see that it's specifically looking for `nextval('message_id_seq'::regclass)` as a default value.
<img width="1090" alt="screen shot 2017-08-02 at 12 34 10 pm" src="https://user-images.githubusercontent.com/24230076/28891602-eec92c0e-777f-11e7-9bf2-b411f6f9a370.png">